tests/tests.js

```diff
@@ -810,6 +810,17 @@ describe('Convert Lat/Lon to USNG', function(){
   });
 });
 describe('UPS Conversions', () => {
+  describe('isInUPSSpace', () => {
+    it('North pole', () => {
+      chai.expect(converter.isInUPSSpace(85)).to.equal(true)
+    })
+    it('South pole', () => {
+      chai.expect(converter.isInUPSSpace(-81)).to.equal(true)
+    })
+    it('Not UPS Space', () => {
+      chai.expect(converter.isInUPSSpace(35)).to.equal(false)
```

Comment:
:pencil2: Just to be sure, could you also add a test to ensure that `false` is returned for a negative values greater than -80?